DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>flake8 errors for this cell:</p>

<pre class="ql-syntax" spellcheck="false">pycodestyle INFO: 2:57: W291 trailing whitespace
pycodestyle INFO: 4:58: W291 trailing whitespace
pycodestyle INFO: 6:38: E231 missing whitespace after ','
pycodestyle INFO: 7:38: E231 missing whitespace after ','
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>